channels/us.m3u

```diff
@@ -550,17 +550,17 @@ https://liveproduseast.global.ssl.fastly.net/us/Channel-USTV-AWS-virginia-1/Sour
 https://bloomberg-bloombergtv-1-gb.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="Bluegrass4U.us" tvg-name="Bluegrass 4U" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/nm3N3aw.png" group-title="",Bluegrass 4U
 https://59d39900ebfb8.streamlock.net/blugrassmusic/blugrassmusic/playlist.m3u8
-#EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Bon Appetit
+#EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",Bon Appetit
 https://bonappetit-roku-us.amagi.tv/hls/amagi_hls_data_condenast-bonappetitroku/CDN/playlist.m3u8
 #EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="Cooking",Bon Appetit (1080p)
 https://bonappetit-samsung.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Bon Appetit
+#EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",Bon Appetit
 https://condenast-bonappetit-vizio.amagi.tv/playlist480p.m3u8
 #EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="https://od.lk/s/MF8yMzE5OTIyMjBf/Redbox_512x512.png" group-title="",Bon Appetit
 https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxbonappetit/CDN/playlist.m3u8
 #EXTINF:-1 tvg-id="BonanzaSTIRR.us" tvg-name="Bonanza (STIRR)" tvg-country="US" tvg-language="English" tvg-logo="https://komonews.com/resources/media2/3x1/full/119/center/90/3385dd03-f70f-42e3-b1e6-98b8b5078a8c-small3x1_stirr_0120_epg_Bonanza_1920x1080.png?cb=c81e728d9d4c2f636f067f89cc14862c" group-title="Classic",Bonanza (STIRR) (720p)
 https://dai.google.com/linear/hls/event/LeVr-Z0_Q4qMDdXx7zr22w/master.m3u8
-#EXTINF:-1 tvg-id="Bonappetit.us" tvg-name="Bonappetit" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Bonappetit
+#EXTINF:-1 tvg-id="Bonappetit.us" tvg-name="Bonappetit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",Bon Appetit
```

Comment:
I remember you talking about this channel. Watch out for the tvg-id and tvg-name, as they doesn't match the ones above.

Reply:
Okay, should I change that? I was staying away from that area because it affects the epg?

Reply:
Since it isn't a TV channel to itself, you're safe to modify that part.

Reply:
I've seen that the tvg-name was modified, not the tvg-id.

Reply:
>  #EXTINF:-1 tvg-id="Bonappetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",Bon Appetit

Change : 

> #EXTINF:-1 tvg-id="BonAppetit.us" tvg-name="Bon Appetit" tvg-country="US" tvg-language="" tvg-logo="https://i.imgur.com/YhFFxlE.png" group-title="",Bon Appetit